src/components/Scorecards/ScorecardsServicePage/ScorecardsServiceHeader.tsx

```diff
@@ -87,7 +88,11 @@ export const ScorecardServiceHeader = ({
           </Box>
           {scorecard.description && (
             <Box mb={1}>
-              <Typography>{scorecard.description}</Typography>
+            <Truncated
+              text={scorecard.description || ''}
```

Comment:
```suggestion
              text={scorecard.description ?? ''}
```
but why not just allow the `text` field to be nullish and handle it internally to the component (DRY)?